query-creator/create-query/app.js

```diff
@@ -34,75 +140,7 @@ exports.lambdaHandler = async (event, context) => {
     // get the post body
     const body = request.getBody(event);
     request.validateRequestBody(body);
-    const { json, jwt } = body;
-    const { query, learnersApiUrl, user } = json;
-    let { email } = user;
-
-    const portalUrl = learnersApiUrl.match(/(.*)\/api\/v[0-9]+/)[1];
-
-    const authDomain = portalToAuthDomainMap[portalUrl] || portalUrl;
-
-    const tokenServiceJwt = await request.getTokenServiceJwt(portalUrl, jwt);
-
-    const resource = await tokenService.findOrCreateResource(tokenServiceJwt, tokenServiceEnv, email, portalUrl);
-    const workgroupName = await aws.ensureWorkgroup(resource, user);
-
-    const queryIdsPerRunnable = await aws.fetchAndUploadLearnerData(jwt, query, learnersApiUrl, narrowLearners);
-
-    const sqlOutput = [];
-
-    const doLearnerAnswerReporting = async (runnableUrl) => {
-      const queryId = queryIdsPerRunnable[runnableUrl];
-
-      // get and denormalize the resource (activity or sequence) from Firebase
-      let resource;
-      let denormalizedResource;
-      try {
-        resource = await firebase.getResource(runnableUrl, reportServiceSource);
-        denormalizedResource = firebase.denormalizeResource(resource);
-
-        // upload the denormalized resource to s3 and tie it to the workgroup
-        await aws.uploadDenormalizedResource(queryId, denormalizedResource);
-      } catch (err) {
-        // no valid resource, we will attempt to create a usage report with just the learner data
-        console.log(err);
-      }
-
-      // generate the sql for the query
-      const sql = aws.generateSQL(queryId, resource, denormalizedResource, usageReport, runnableUrl, authDomain, reportServiceSource);
-
-      if (debugSQL) {
-        sqlOutput.push(`${resource ? `-- id ${resource.id}` : `-- url ${runnableUrl}`}\n${sql}`);
-      } else {
-        // create the athena query in the workgroup
-        await aws.startQueryExecution(sql, workgroupName)
-      }
-    }
-
-    const doLearnerLogReporting= async (runnableUrl) => {
-      const queryId = queryIdsPerRunnable[runnableUrl];
-
-      // generate the sql for the query
-      const sql = narrowLearners
-        ? aws.generateNarrowLogSQL(queryId, runnableUrl, authDomain, reportServiceSource)
-        : aws.generateLogSQL(queryId, runnableUrl, authDomain, reportServiceSource);
-
-      if (debugSQL) {
-        sqlOutput.push(`-- url ${runnableUrl}\n${sql}`);
-      } else {
-        // create the athena query in the workgroup
-        await aws.startQueryExecution(sql, workgroupName)
-      }
-    };
-
-    for (const runnableUrl in queryIdsPerRunnable) {
-      if(useLogs) {
-        await doLearnerLogReporting(runnableUrl);
-      }
-      else {
-        await doLearnerAnswerReporting(runnableUrl);
-      }
-    }
+    const {sqlOutput, portalUrl} = body.json.type === "users" ? await userReport(body, tokenServiceEnv, debugSQL) : await learnersReport(params, body, tokenServiceEnv, debugSQL, reportServiceSource);
```

Comment:
This is a long line. How about this formating:

```suggestion
    const {sqlOutput, portalUrl} = body.json.type === "users" 
      ? await userReport(body, tokenServiceEnv, debugSQL) 
      : await learnersReport(params, body, tokenServiceEnv, debugSQL, reportServiceSource);
```

Reply:
Done.  Updated in commit: https://github.com/concord-consortium/report-service/pull/159/commits/910cc6a8ca2b1a81e2b2a5e96b3574e171f9f7e5